circle.yml

```diff
@@ -130,7 +130,12 @@ scan_build_run: &scan_build_run
     - *export_logs
     - run:
         name: Check scan-build results
-        command: test ! -d ~/libratbag/build/meson-logs/scanbuild || test $(find ~/libratbag/build/meson-logs/scanbuild -maxdepth 0 ! -empty -exec echo "not empty" \; | wc -l) -eq 0 || (echo "Check scan-build results" && false)
+        command: |
+          test ! -d ~/libratbag/build/meson-logs/scanbuild || (for r in ~/libratbag/build/meson-logs/scanbuild/*/report*.html
+          do
+            line=$(grep -Po 'a href="#EndPath">line \K(\d)+' $r)
+            grep -o "LN${line}\">.*ignore_clang_sa_" $r || exit 1
+          done)
```

Comment:
I think we're at the point where we should move this into a script as part of the git repo and just have circleCI supply the directory. Much easier to debug this way. Other than that, it looks good though.